physical/tvf.go

```diff
@@ -187,6 +187,32 @@ func (node *TableValuedFunction) Materialize(ctx context.Context, matCtx *Materi
 
 		return tvf.NewTumble(matSource, timeField, matWindowLength, matWindowOffset), nil
 
+	case "zero":
```

Comment:
yes indeed, this shouldn't be a tvf, but the default. (and be returned in the todo in the range tvf)